sdk/inc/azure/core/az_version.h

```diff
@@ -17,19 +17,19 @@
 
 /// The version in string format used for telemetry following the `semver.org` standard
 /// (https://semver.org).
-#define AZ_SDK_VERSION_STRING "1.1.0"
+#define AZ_SDK_VERSION_STRING "1.2.0-beta.1"
 
 /// Major numeric identifier.
 #define AZ_SDK_VERSION_MAJOR 1
 
 /// Minor numeric identifier.
-#define AZ_SDK_VERSION_MINOR 1
+#define AZ_SDK_VERSION_MINOR 2
 
 /// Patch numeric identifier.
 #define AZ_SDK_VERSION_PATCH 0
 
 /// Optional pre-release identifier. SDK is in a pre-release state when present.
-#define AZ_SDK_VERSION_PRERELEASE
-#undef AZ_SDK_VERSION_PRERELEASE
+#define AZ_SDK_VERSION_PRERELEASE "beta.1"
+#define AZ_SDK_VERSION_PRERELEASE "beta.1"
```

Comment:
Aww, the auto-updater ends up defining this twice.

Either we need to stop using `#define/undef` like this, OR have the auto-updater know to undo it.

See previous conversation here: https://github.com/Azure/azure-sdk-for-c/pull/1336#discussion_r492428630

cc @danieljurek, @antkmsft